Parsedown.php

```diff
@@ -1453,8 +1458,19 @@ protected function elements(array $Elements)
         return $markup;
     }
 
+    #
+    # - [x] AND - [ ]
+    protected function checkbox(&$text,&$attributes)
```

Comment:
Perhaps have this method build the entire `li` and return that, to avoid returning multiple pieces of data/using references?